app/models/project_creation.rb

```diff
@@ -0,0 +1,9 @@
+module ProjectCreator
+  def self.generate(contractor, tribe, params)
+    params.merge({
+      total: tribe.total,
+      contractor: contractor,
+      developers: tribe.developers
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.